include/open62541/config.h.in

```diff
@@ -27,6 +27,7 @@
 #cmakedefine UA_ENABLE_NODEMANAGEMENT
 #cmakedefine UA_ENABLE_SUBSCRIPTIONS
 #cmakedefine UA_ENABLE_PUBSUB
+#cmakedefine UA_ENABLE_PUBSUB_CONFIG
```

Comment:
I think UA_ENABLE_PUBSUB_FILE_CONFIG would add more clarity. How do you think about that @jpfr.